### DISCUSSION TITLE
INSTALL STEP BY STEP

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#879] [Q&A] [answered]
Hi. I read https://github.com/OpenRadioss/OpenRadioss/blob/main/HOWTO.md 
 I would like to ask if for example I want instal openradioss on Linux , why I need WSL  ? could understand if I was installing on Windows but I don't understand what WSL is for when installing on LINUX

**Comment by MarianBulla** [comment, marked as answer]
Hi, 
If you want to install OpenRadioss on Linux, you don't need to install WSL.
WSL (= Windows Subsystem for Linux) is just a Linux on top of Windows.
We suggested it, if you have a windows OS but want to use the linux compiler.
Now, we also support Windows directly, using the precompiled executlables or (using cygwin) also the sources, where you can provide your developments, if you want.
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by CinekPoland** [reply]
Hi. Thank you for your reply. It helped me very much.  I installed newest visual studio 2022,  IntelONEAPI basee toolkit, intelONEAPI  HPC  toolkit , and microsoft MPI, I was traing run cell phone drop test and bumper simulation, and I got this error how to fix it ? 
![pytanie](https://user-images.githubusercontent.com/119892611/207988813-9d71731e-2c45-4313-90b7-cb79b142afba.jpg)
